cmake/script/CoverageInclude.cmake.in

```diff
@@ -12,7 +12,7 @@ endif()
 
 # COV_TOOL is used to replace a placeholder.
 configure_file(
-  cmake/cov_tool_wrapper.sh.in ${CMAKE_CURRENT_LIST_DIR}/cov_tool_wrapper.sh
+  ${CMAKE_CURRENT_LIST_DIR}/cov_tool_wrapper.sh.in ${CMAKE_CURRENT_LIST_DIR}/cov_tool_wrapper.sh
```

Comment:
Could this not be relative to `${PROJECT_SOURCE_DIR}` instead to avoid the two-step copy/process?

Reply:
With the current design, the required `gcov` or `llvm-cov` tools are searched during the `Coverage{Fuzz}.cmake` script invocation. This two-step processing could potentially be avoided by moving these tool checks into the main buildsystem script. However, there is no reliable way to gate them depending on whether the build configuration is 'Coverage' or not.